src/mixins/Translate.ts

```diff
@@ -0,0 +1,29 @@
+import { Vue, Component } from 'vue-property-decorator';
+import Utils from '@/utils/Utils';
+
+@Component
+export default class MyMixin extends Vue {
+  // region public static methods
+  // endregion
+
+  // region private static methods
+  public translateToNaturalLanguage(text: string): string {
```

Comment:
And maybe we just want to call it translate? Later, it would also be possible to add multi-language support using this function. For me, `translate` would be just fine.

Reply:
Maybe later